### DISCUSSION TITLE
Vimeo-like ante-piracy protection

### DISCUSSION THREAD

**Discussion by xmontero** [videojs/video.js#8441] [General]
There are video-ripper browser add-ons that are able to detect videos on web pages and allow to download them.

Vimeo has an anti-piracy mechanism that allows companies to share private videos and protect against "video ripper add-ons". If you try to do so, a black-image and no audio is shown all along the video duration.

I need to implement an anti-piracy video player to allow a company to share videos with providers but disabling them to rip the video into their hard disk.

Does Video.js provide any kind of such functionality?
Ie: If the mandatory requiremets are "anti-download anti-pirate systems" + "open-source", is Video.Js something I should look at?

**Comment by mister-ben** [comment]
You need DRM for that type of content protection. It's the DRM content decryption module (CDM) that blanks the video. [videojs-contrib-eme](https://github.com/videojs/videojs-contrib-eme) facilitates DRM playback.

**Reply by xmontero** [reply]
Will look at that. Thanks! 👍
